deployment/gke/deployment_manager_configs/cluster.jinja

```diff
@@ -120,6 +120,7 @@ resources:
 # We do this so that if we want to make changes we can delete the existing resource and then recreate it.
 # Updating doesn't work so well because we are limited in what changes GKE's update method supports.
 
+{% if properties['gpu-pool-max-nodes'] > 0 %}
```

Comment:
Does this mean, by default there is no gpu pool and KF deployments done via web-app won't have a gpu_pool? If so, we may want to add a checkbox option in the webapp to allow creation of gpu pool.

Reply:
IIRC, I got an error when I tried setting the node # to 0, but creating the node pool -- it is not happy with a 0-node pool.

Reply:
How about make region a dropdown list and only include region which provide GPUs?

Reply:
I feel when user finished testing and want to start some real workload, they will need GPUs. It would be pain to suddenly figure out GPU not available on their region and have to migrate cluster.